Manifest.toml

```diff
@@ -0,0 +1,262 @@
+# This file is machine-generated - editing it directly is not advised
```

Comment:
Committing `Manifest.toml` into a repo is not suitable for a library package . Please remove it